client/Lykke.AlgoStore.MatchingEngineAdapter.Client/RequestManager.cs

```diff
@@ -44,25 +51,21 @@ public void SetClientAndInstanceId(string clientId, string instanceId)
 
         private void ProcessResponse(IMessageInfo messageInfo)
         {
-            if (messageInfo == null)
-                throw new ArgumentNullException(nameof(messageInfo));
-
             _taskManager.SetResult(messageInfo.Id, messageInfo);
         }
 
-        private void SendAuthRequest()
+        private async void SendAuthRequest()
         {
             var request = new PingRequest { Message = $"{_clientId}_{_instanceId}" };
 
-            var task = MakeRequestAsync(MeaRequestType.Ping, request);
+            var response = (await MakeRequestAsync(MeaRequestType.Ping, request)).Message as PingRequest;
```

Comment:
The method async and void. It means that caller will not wait for the result - fire and forget pattern. Here I see you call MakeRequestAsync which can throw exceptions. Who will catch these exception? And how in Production you will known that there was exception here?
As minimum you should log the exception.

Reply:
The case where MakeRequestAsync throws an exception will never be encountered - the bool _authorizationFailed is only set inside this method, and it will always be false when MakeRequestAsync is invoked.